CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [#6103](https://github.com/rubocop-hq/rubocop/pull/6103): Fix a false positive for `Layout/IndentationWidth` when multiple modifiers are used in a block and a method call is made at end of the block. ([@koic][])
 * [#6084](https://github.com/rubocop-hq/rubocop/issues/6084): Fix `Naming/MemoizedInstanceVariableName` cop to allow methods to have leading underscores. ([@kenman345][])
+* [#6105](https://github.com/rubocop-hq/rubocop/issues/6105): Support `{a,b}` file name globs in `Exclude` and `Include` config. ([@mikeyhew][])
```

Comment:
This is a new notation for `Include` and `Exclude`. Please make `### New features` section in `## master (unreleased)` and move there. This is an awesome extension 🌟